installers/nix/linux.nix

```diff
@@ -24,82 +26,25 @@ let
     mkdir -pv $out/bin/
     cp ${openssl}/bin/openssl $out/bin/
   '';
+
+  installationDhall = writeTextFile {
+    name = "installation.dhall";
+    text = ''
+    { cardano              = "${cardanoPkgs.cardano-sl}"
+    , configFiles          = "${configFiles}"
+    , daedalusFrontend     = "${daedalus_frontend}"
+    }'';
+  };
+
   configFiles = runCommand "cardano-config" {} ''
```

Comment:
`runCommand "cardano-config" { buildInputs = [ installer-generation ]; } ''` will put it into PATH automatically

Reply:
Done!